<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelsPane</class>
 <widget class="QWidget" name="PanelsPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>plainControlCenter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QPushButton" name="backPushButton">
     <property name="text">
      <string> Close</string>
     </property>
     <property name="icon">
      <iconset theme="go-previous">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="panelsLabel">
     <property name="text">
      <string>Panels (max 4)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="panelsListWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="addPanelPushButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removePanelPushButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="appletsLabel">
     <property name="text">
      <string>Applets</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="availableAppletsListWidget">
       <item>
        <property name="text">
         <string>appmenu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>windowlist</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>spacer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>workspaces</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>volume</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kblayout</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>datetime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>splitter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>usermenu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>localipv4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mpris</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>launcher</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>battery</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sni</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>clioutput</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="addAppletPushButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeAppletPushButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveUpPushButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>⬆</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveDownPushButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>⬇</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="configureAppletPushButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="preferences-system"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="enabledAppletsListWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="launcherIconSizeLabel">
       <property name="text">
        <string>Launcher icon size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="launcherIconSizeSpinBox">
       <property name="maximum">
        <number>256</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="expandCheckBox">
       <property name="text">
        <string>Expand panel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="xOffsetLabel">
         <property name="text">
          <string>Axis shift</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="shiftSpinBox">
         <property name="maximum">
          <number>16777215</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="thicknessLabel">
         <property name="text">
          <string>Panel thickness</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="thicknessSpinBox">
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="spacingLabel">
         <property name="text">
          <string>Spacing between applets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spacingSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="locationLabel">
         <property name="text">
          <string>Panel location</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QRadioButton" name="bottomRadioButton">
           <property name="text">
            <string>Bottom</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="topRadioButton">
           <property name="text">
            <string>Top</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="leftRadioButton">
           <property name="text">
            <string>Left</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="rightRadioButton">
           <property name="text">
            <string>Right</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="opacityLabel">
       <property name="text">
        <string>Panel opacity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="opacitySpinBox">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="revertPushButton">
     <property name="text">
      <string>Revert selected panel configuration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="savePushButton">
     <property name="text">
      <string>Save selected panel configuration</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
